src/components/time-picker/time-picker.tsx

```diff
@@ -847,7 +861,10 @@ export class TimePicker {
           <span class={CSS.delimiter}>{this.localizedSecondSuffix}</span>
         )}
         {showMeridiem && (
-          <div class={CSS.column} role="group">
+          <div
+            class={{ [CSS.column]: true, [CSS.meridiemStart]: this.getMeridiemOrder() === 0 }}
```

Comment:
Since `getMeridiemOrder` is mostly used to determine if it's at the start or not, why not refactor it to something like `isMeridiemAtStart(): boolean`? Disregard this if you think the order number may be used in the near future.

Reply:
it's better to keep is as `getMeridiemOrder( ) ` considering it might be used in future or may be move it to utils ?